hack/test/docker/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM gcr.io/iguazio/golang:1.14
```

Comment:
we work with 1.16 i believe

Reply:
you sure, go mod states 1.14, should I update that?

Reply:
Saving this change for a future PR